src/modules/trade/helpers/generate-trade.js

```diff
@@ -23,14 +23,17 @@ export const generateTrade = memoize((market, outcome, outcomeTradeInProgress, o
 
   const side = (outcomeTradeInProgress && outcomeTradeInProgress.side) || TRANSACTIONS_TYPES.BUY
   const numShares = (outcomeTradeInProgress && outcomeTradeInProgress.numShares) || null
+  const sharesFilled = (outcomeTradeInProgress && outcomeTradeInProgress.sharesFilled) || null
   const limitPrice = (outcomeTradeInProgress && outcomeTradeInProgress.limitPrice) || null
   const totalFee = (outcomeTradeInProgress && outcomeTradeInProgress.totalFee) || 0
```

Comment:
`totalFee`, `feePercent`, `gasFeesRealEth`, and `totalCost` default to JS numeric 0; as with other numerical quantities, these should generally be either strings or BigNumbers.  I suggest making the default values here also strings or BigNumbers (whichever the variables themselves are) for consistency.